theano/tensor/tests/test_basic.py

```diff
@@ -2159,6 +2164,7 @@ def fixed_first_input_iv(x):
     op=inplace.conj_inplace,
     expected=numpy.conj,
     good=_good_broadcast_unary_normal,
+    grad=None,
```

Comment:
This `grad` parameter that you pass to `makeBroadcastTester` is a `kwarg` which is simply passed on to `makeTester` that sets the default value of `grad` to `None`. Instead, it is better you delete all these instances. If that works, that is a better fix than passing `None`.